internal/web3ext/web3ext.go

```diff
@@ -628,7 +628,11 @@ web3._extend({
                new web3._extend.Property({
                        name: 'leader',
                        getter: 'raft_leader'
-               })
+               }),
+               new web3._extend.Property({
+                       name: 'cluster',
```

Comment:
Because this new property does not contain information about the current node (which would then constitute a full "cluster"), in this current implementation it might be better named as "peers" instead?

E.g. currently for a cluster of size three it reports that it contains only two nodes, which could be confusing:

```
> raft.cluster.length
2
```

It seems like we should probably keep the name "cluster", but also include the current node's `RaftNodeInfo.Address` alongside the `RaftNodeInfo.PeerAddresses` that we're reporting back to the client.